src/cmd/singularity/cli/sif.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/singularityware/singularity/src/pkg/buildcfg"
+	"github.com/singularityware/singularity/src/pkg/sylog"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	sifCmd.Flags().SetInterspersed(false)
+	singularityCmd.AddCommand(sifCmd)
+}
+
+var sifCmd = &cobra.Command{
+	Use:    "sif COMMAND OPTION FILE",
+	Hidden: true,
```

Comment:
Question... should the `singularity sif` command be hidden?

Reply:
I think `singularity sif` is fine not being hidden. We were talking about `singularity sbuild` being hidden, but we did away with that idea anyway.

Reply:
Actually, if we're not going to hide it there is one more change that should be made. Can you add appropriate Help, Usage, Example, etc... strings to the command so that cobra can autogenerate the help for this command? Thanks ed